.github/workflows/edit.yml

```diff
@@ -0,0 +1,16 @@
+name: edit
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  labels:
+
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: yogevbd/enforce-label-action@2.1.0
+        with:
+          REQUIRED_LABELS_ANY: "bug,enhancement,feature,deprecated,major,skip-changelog"
+          REQUIRED_LABELS_ANY_DESCRIPTION: "Select at least one label: bug,enhancement,feature,deprecated,major,skip-changelog"
```

Comment:
By specifying the list here, you've created a double maintenance point where docs could get out of sync with the code. The action's default description avoids this by defaulting to `Please select one of the required labels for this PR: ${requiredLabelsAny}`